catalog/nsili/catalog-nsili-endpoint/src/main/resources/OSGI-INF/metatype/org.codice.alliance.nsili.endpoint.emailSender.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD name="NSILI EmailSender" id="org.codice.alliance.nsili.endpoint.EmailSender">
+        <AD description="Set the hostname of the mail server."
+            name="Mail Server" id="mailHost" required="true" type="String"
+            default="connexta-com.mail.protection.outlook.com"/>
```

Comment:
Switched to just a blank `""` string instead of the residual `connexta-com.mail.protection.outlook.com`